File: src/net/Defracted/AdminUtils/commands/AdminUtils.java
package net.Defracted.AdminUtils.commands;

import net.Defracted.AdminUtils.Main;
import net.Defracted.AdminUtils.util.Formatters;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class AdminUtils implements CommandExecutor {
    static Main plugin;

    public AdminUtils(Main instance) {
        plugin = instance;
    }

    public boolean onCommand(CommandSender sender, Command cmd, String label, String[] args) {
        // /adminutiles
        if (cmd.getName().equalsIgnoreCase("adminutils")) {
            // /adminutils reload
            if (args[0].equalsIgnoreCase("reload")) {
                plugin.reloadConfig();
                sender.sendMessage(Formatters.chat("&aAdminUtils reloaded."));
                return true;
            }

            return true;
        }

        return false;
    }
}

File: src/main/kotlin/net/defracted/adminutils/commands/AdminUtils.kt
package net.defracted.adminutils.commands

import net.defracted.adminutils.Main
import net.defracted.adminutils.util.Formatters.chat

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender


class AdminUtils(private val plugin: Main) : CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, label: String, args: Array<String>): Boolean {
        // /adminutiles
        if (cmd.getName().equals("adminutils", ignoreCase = true)) {
            // /adminutils reload
            if (args[0].equals("reload", ignoreCase = true)) {
                plugin.reloadConfig()
                sender.sendMessage(chat("&aAdminUtils reloaded."))
                return true
            }
            return true
        }
        return false
    }
}